docs/Preparing Project Launch/WPL Preparation/landing-page/7-PPL-landing-page.md

```
@@ -0,0 +1,45 @@
+
+<p align="center">
+  <img src="../../../images/logo-etsii.jpe" alt="Logo ETSII" width="400">
+</p>
+
+
+
+## Historial de Versiones
+
+| Versión | Fecha | Cambio principal |
+|---------|-------|------------------|
+| 1.0.0 | 13/05/2026 | Versión inicial|
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)